Source/SwiftLintFramework/Rules/Lint/TestCaseAccessibilityRuleExamples.swift

```diff
@@ -14,30 +14,6 @@ internal struct TestCaseAccessibilityRuleExamples {
                 return nestedMember
             }
 
-            override static func setUp() {
-                super.setUp()
```

Comment:
Better to remove the `super.method()` lines only to still have explicit tests for these methods.

Reply:
Good idea 👍